### ISSUE TITLE
deleteBy creates a syntax error 

### ISSUE THREAD

**Issue by madflow** [mlipscombe/postgraphile-plugin-nested-mutations#31]
Consider the following relations:

```SQL
CREATE TABLE a (a_id SERIAL PRIMARY KEY, name TEXT);
CREATE TABLE b (b_id SERIAL PRIMARY KEY, name TEXT);
CREATE TABLE a_b (a_id INT REFERENCES a(a_id), b_id INT REFERENCES b(b_id), PRIMARY KEY (a_id, b_id));
```

When run through a mutation:

```
mutation MyMutation {
  createA(
    input: { a: { aBsUsingAId: { deleteByAIdAndBId: { a_id: 10, b_id: 10 } } } }
  ) {
    clientMutationId
  }
}
```

it yields:

```SQL
db_1       |                insert into "public"."a"
db_1       |                  default values returning *
db_1       | 2020-01-22 07:45:30.397 UTC [125] ERROR:  syntax error at or near ")" at character 102
db_1       | 2020-01-22 07:45:30.397 UTC [125] STATEMENT:
db_1       |                delete from "public"."a_b"
db_1       |                where
db_1       |                    "a_id" = $1
db_1       |                  ) and (
db_1       |                    "b_id" = $2
```

This is addressed here https://github.com/mlipscombe/postgraphile-plugin-nested-mutations/pull/24 I believe. In this issue I tried to provide a test case (since the PR is missing tests.)

**Comment by frisi** [comment]
i can confirm the bug and also that @phryneas PR https://github.com/mlipscombe/postgraphile-plugin-nested-mutations/pull/24 solves the issues.

**Comment by AndresGnu** [comment]
Hello I confirm this issue

I have:
```gql
mutation{
  updateTableById(,input:{
    id:"BkBank",
    patch:{
      appPermissionsUsingId:{
         deleteByTableIdAndActionAndGroup:{
          group: "fastcont-sellers",
          action: C,
          tableId:"BkBank"
        }
      }
    }
  }){
    table{
      id
    }
  }
}
```
Response is:
```gql
{
  "errors": [
    {
      "message": "syntax error at or near \")\"",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "updateTableById"
      ]
    }
  ],
  "data": {
    "updateTableById": null
  }
}
```

**Comment by mlipscombe** [comment]
This is fixed by merging @phryneas PR #24.  I'll be pushing a new release to npm later today.
